torch/distributed/tensor/parallel/fsdp.py

```diff
@@ -320,19 +320,36 @@ class DTensorExtensions(FSDPExtensions):
     This is the implementation for FSDPExtensions defined in
     https://github.com/pytorch/pytorch/blob/main/torch/distributed/fsdp/_fsdp_extensions.py
     """
+    def __init__(self, device_handle) -> None:
+        super().__init__()
+        self.compute_stream = None
+        self.device_handle = device_handle
 
     def pre_flatten_transform(
         self,
         tensor: torch.Tensor,
     ) -> Tuple[torch.Tensor, Optional[Any]]:
         return _flatten_tensor(tensor)
 
+    @torch._dynamo.disable
```

Comment:
Will this matter for @voznesenskym 's work tracing FSDP 🤔

Reply:
I tried remove this and dynamo complains about a stream related issue. I guess once @voznesenskym enable the tracing FSDP we can visit this to ensure the 2D parallelism don't trigger graph break